samples/kvsWebRTCClientMaster.c

```diff
@@ -25,7 +27,24 @@ INT32 main(INT32 argc, CHAR* argv[])
     pChannelName = argc > 1 ? argv[1] : SAMPLE_CHANNEL_NAME;
 #endif
 
-    CHK_STATUS(createSampleConfiguration(pChannelName, SIGNALING_CHANNEL_ROLE_TYPE_MASTER, TRUE, TRUE, logLevel, &pSampleConfiguration));
```

Comment:
Would suggest keeping this here and then setting the pSampleConfiguration codec related params after we read the args.

Reply:
Done